projectgenerator/gui/export.py

```diff
@@ -46,9 +50,32 @@ def __init__(self, base_config, parent=None):
         self.base_configuration = base_config
         self.restore_configuration()
 
+        self.validators = Validators()
+        regexp = QRegExp("[a-zA-Z_0-9]+") # Non empty string
```

Comment:
I think for hostnames there are other characters allowed (or even expected) like `.` (ip addresses), `-` etc. Is this properly handled here? Maybe it's better to really just check for anything else than empty string?